docs/source/contents/4_draft_mapping.rst

```diff
@@ -841,7 +855,7 @@ When there is no :code:`mods:name/@valueURI`, use the string literal from :code:
     <https://example.org/objects/1>
         relators:cre "Daniel, Charles R. (Charlie), Jr., 1930-" .
 
-If there is a :code:`mods:name/valueURI` but it's empty, use the string literal instead:
+If there is a :code:`name/valueURI` but it's empty, use the string literal instead:
```

Comment:
I don't think we've represented this appropriately since valueURI is an attribute on name. Perhaps change it to :code: `name[@valueURI]`?

Reply:
Great catch, good question, and this is something that we should clarify:

`name/@valueURI` refers to the value of the attribute; e.g.
```xml
<name valueURI="http://www.something.net/names">Bridger</name>
```
vs `name[@valueURI]` which refers to the name element; i.e. select all `name` elements with a `valueURI` attribute. I think this particular part of our document is saying, "If there is a valueURI attibute on the name element but the attribute is empty, use the string value of the element." It should still have an `@`!!!

Reply:
Yes, I was not reading closely enough. We should have an @, but the rest is fine.

Reply:
Thanks for adding the attribute symbol.

Reply:
Thank you for catching it!